### ISSUE TITLE
consider upgrading to `bcrypt.c` v1.58

### ISSUE THREAD

**Issue by t1m0t** [riverrun/bcrypt_elixir#45]
Here there more recent source file of [`bcrypt.c`](https://github.com/openbsd/src/blob/master/lib/libc/crypt/bcrypt.c).

Would it be worth considering an upgrade of this source file?

**Comment by riverrun** [comment]
Yes, I just looked at the changes, and they do not affect the nif implementation.

The changes are to a function that measures the system's performance, and we do not include this function in this library's implementation.

Thanks for raising the issue, and sorry for not getting back to you earlier.
